### PR TITLE
Redo shinytests to capture changes in the metrics (constraints).

### PR DESCRIPTION
**Pull request by cjcallag**
Redid the tests done with **{shinytest}** to capture new metrics (constraint). Test was recorded using a seed and the following code:

```
shinytest::recordTest(app = "inst/shiny-snExplorer/", seed = 123)
```

For future reference, do not check box for "Save inputs that do not have a binding".

### CODE REVIEW

File: tests/testthat/test-shiny-snExplorer.R [hunk 1]
```diff
@@ -1,12 +1,13 @@
+library(shinytest)
+library(testthat)
 context("app-file")
 # This file is for testing the applications in the inst/ directory.
 
-library(shinytest)
-
 test_that("snExplorer works", {
     skip_on_cran()
     skip_on_ci() # something's off on system dependencies
```
Comment (knapply) [inline tests/testthat/test-shiny-snExplorer.R:8]: Do we dare put test on through CI?

We're still in a branch, so this is the best timing we'll get.

**Comment by knapply** [comment]
> For future reference, do not check box for "Save inputs that do not have a binding".

This may be worth putting in an issue for far-future reference.

File: tests/testthat/test-shiny-snExplorer.R (hunk 1 above)
Comment (cjcallag) [reply tests/testthat/test-shiny-snExplorer.R:8]: Do it! Isn't it the right time?

**Comment by cjcallag** [comment]
Adding:
- **{shinytest}** tests
- Citation
- CI integration
- **curl** to `DESCRIPTION`

**Pull request merged**